src/config/backpack/basset.php

```diff
@@ -1,6 +1,9 @@
 <?php
 
 return [
+    // development mode, assets will not be internalized
+    'dev_mode' => false,
```

Comment:
We should probably have an ENV variable for this. That way, most devs wouldn't need to publish the config file at all. Do you have a better name than this?

```suggestion
    'dev_mode' => env('BASSET_DEV_MODE', false),
```

Reply:
In the beginning I thought about using `APP_ENV`, so `env('APP_ENV') === 'local'`, but it shouldn't be dependent to the `APP_ENV`, should it? I better agree with your var `BASSET_DEV_MODE` 👌